### DISCUSSION TITLE
Metric to scale webserver using keda

### DISCUSSION THREAD

**Discussion by deepak4babu** [apache/airflow#42470] [Q&A]
Is there any airflow webserver metric to autoscale webserver using keda? 

We make many api requests to trigger dags remotely and also use airflow web UI regularly. At times, API requests are timing out. Hence we are looking for a metric to scale airflow webserver automatically.

**Comment by deepak4babu** [comment]
@potiuk Sorry to tag you out of the blue. Can you please help with this (or) tag relevant folks who can?

**Reply by JoaVirtudes19** [reply]
I recently added Horizontal Pod Autoscaler (HPA). Take a look at my [PR](https://github.com/apache/airflow/pull/41955). If you enable HPA the webserver scale up and down automatically based on the metric you provide in configuration.

**Reply by JoaVirtudes19** [reply]
Although you will have to wait for the next version of the helm chart

**Reply by deepak4babu** [reply]
@JoaVirtudes19 Thanks. Actually im looking for the right metric to scale webserver. Can I know, why you have chosen CPU as a metric to scale horizontally
